sandy/core/cov.py

```diff
@@ -1638,8 +1907,15 @@ def random_ctg_cov(index, stdmin=0.0, stdmax=1.0, correlations=True, seed=None):
     return pd.DataFrame(cov, index=index, columns=index)
 
 
-def get_eig(cov, sort=True):
-    E, V = scipy.linalg.eig(cov)
+def get_eig(cov, sparse=False, sort=True):

```

Comment:
Last comment,  I think there's no need anymore to have a function `get_eig`.
This algorithm should be merged in method `eig` of `CategoryCov` (maybe being called `get_eigs`).

What do you think?

Reply:
In my opinion is better to merge the algorithm in `eig` instead of create a new method.

Reply:
Yes, in `eig`...I just don't like the name...I think calling it `get_eigs` is better